synapse/rest/admin/rooms.py

```diff
@@ -253,6 +253,43 @@ async def on_GET(self, request):
         return 200, response
 
 
+class RoomRestServlet(RestServlet):
+    """Get room details.
+    This needs user to have administrator access in Synapse.
+    TODO: Return power level for each user
+    TODO: Add on_POST to allow room creation without joining the room
+
+    GET /_synapse/admin/v1/rooms/<room_id>
+
+    returns:
+        200 OK with users if success otherwise an error.
```

Comment:
I don't follow this part of the comment, I'm unsure if it adds much to be honest and can probably just be removed.